### PR TITLE
Cloud setup

### PR DESCRIPTION
**Pull request by victorvarza**
- Review documentation
- Enhance scripts

### CODE REVIEW

File: src/aws/docker/main.tf [hunk 1]
```diff
@@ -3,10 +3,10 @@ provider "aws" {
 }
 
 locals {
-  # adjust cluster lists based on your needs
+  # adjust vm lists based on your needs
   dockerhost = {
     test1 = {
-      allowed_cidr_blocks = "0.0.0.0/0",
+      allowed_cidr_blocks = "188.26.14.158/32",
```
Comment (adriananeci) [inline src/aws/docker/main.tf]: Roll back to all

File: src/aws/k8s/main.tf [hunk 2]
```diff
@@ -5,7 +5,7 @@ provider "aws" {
 locals {
   clusters = {
     test1 = {
-      allowed_cidr_blocks = "0.0.0.0/0",
+      allowed_cidr_blocks = "188.26.14.158/32",
```
Comment (adriananeci) [inline src/aws/k8s/main.tf]: Roll back to all

File: src/utils/tools/get-lab-config.sh [hunk 3]
```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/env bash
 
-USER="$1"
-TOKEN="$2"
-
 if [[ "$USER" == "" || "$TOKEN" == "" ]]; then
-    echo "usage: $0 STUD_X GIT_TOKEN"
+    echo "USER and TOKEN environment variables not set."
```
Comment (adriananeci) [inline src/utils/tools/get-lab-config.sh:4]: Unde se setează astea doua env vars?

**Review by adriananeci — CHANGES_REQUESTED** [review]

File: src/utils/tools/get-lab-config.sh (hunk 3 above)
Comment (victorvarza) [reply src/utils/tools/get-lab-config.sh:4]: In `_K8s lab - init` vor fi informatiile.

**Review by adriananeci — APPROVED** [review]

**Pull request merged**